packages/api/src/useFetch.ts

```diff
@@ -18,7 +18,7 @@ const useFetch = <T extends TSocketEndpointNames>(name: T, ...props: TSocketAcce
 
     return useQuery<TSocketResponseData<T>, unknown>(getQueryKeys(name, payload), () => send(name, payload), {
         ...options,
-        enabled: options?.enabled && is_authorized,
+        enabled: is_authorized && (options?.enabled || true),
```

Comment:
Refactoring `enabled` condition with proper [one](https://github.com/binary-com/deriv-app/pull/9913/files#diff-4685fa33037a4264e6c9fb1e9adfca1f0559bdde8591c2fc21bea7314cfa0b91).
Wrong condition was one of the cause of the issue.

Reply:
Isn't this condition `is_authorized && (options?.enabled || true)` same as writing `is_authorized && true` || `is_authorized` ?

Reply:
I think this is what you want :thinking: 
```suggestion
        enabled: options?.enabled ? is_authorized : false
```

Reply:
I think it should be
```suggestion
        enabled: is_authorized && (options?.enabled ?? true),
```

Reply:
yes guys it should be `enabled: is_authorized && (options?.enabled ?? true)`